### DISCUSSION TITLE
Error while signing up

### DISCUSSION THREAD

**Discussion by a-sajjad72** [vercel/community#3428] [Help] [answered]
### Summary

I attempted to sign up for vercel today. I choose signup with email. entered email then a window appears to verify my email for some code name, received an email verfification, clicked on "Verify" button and when browse to the sign up windows there something written  "user account is blocked". when retry to sign up it says email already exits. then try to login it says "we are unable to complete you signup."

### Steps to Reproduce

Goto  sign up
select Sign up with email
enter email address
verify the email address sent by vercel
and then you see an error there.

**Comment by amyegan** [comment, marked as answer]
Due to the sensitive nature of account details, we handle this type of issue via email. Please send a message to registration@vercel.com and include the email and username for the account. Vercel employs multiple layers of security and verification. Providing additional information about your account and the project you are deploying can help accelerate this process.

**Reply by quarkhadron** [reply]
@amyegan Seem like I accidentally exceeded the bandwidth of hobby plan. I have removed that project. May I ask for steps to get the account unblock? 

Below is my question

https://github.com/orgs/vercel/discussions/3427

**Reply by a-sajjad72** [reply]
sorry but i was not deploying any project. i was just creating an account for vercel. after creation  of account i will be able to deploy projects.
